solr/contrib/ltr/src/java/org/apache/solr/ltr/FeatureLogger.java

```diff
@@ -192,6 +192,33 @@ public String makeFeatureVector(LTRScoringQuery.FeatureInfo[] featuresInfo) {
       return features;
     }
 
+    /*
```

Comment:
Yeah, I'd considered the same but they don't all live in the same package now (`ltr` and `ltr.feature`) and it's also plausible that folk's proprietary test code (living in different package) might wish to use these utility functions.